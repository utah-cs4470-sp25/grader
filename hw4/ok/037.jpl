assert a.n9999999, ""
