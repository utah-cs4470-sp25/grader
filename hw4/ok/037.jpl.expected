(AssertCmd (DotExpr (VarExpr a) n9999999) "")
